packages/osl-db-migrator-tool-image/resources/modules/osl-db-migrator-tool/postgresql/configure

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+set -e
+
+SOURCES_DIR=/tmp/artifacts
+SCRIPT_DIR=$(dirname "${0}")
+ADDED_DIR="${SCRIPT_DIR}"/added
+
+mkdir -p "${KOGITO_HOME}"/bin/quarkus-app
+unzip "${SOURCES_DIR}"/db-migrator-tool-image-build.zip -d "${KOGITO_HOME}"/bin/quarkus-app/
```

Comment:
```suggestion
mkdir -p "${KOGITO_HOME}"/bin/
unzip "${SOURCES_DIR}"/db-migrator-tool-image-build.zip -d "${KOGITO_HOME}"/bin/
```

The kogito-app-launch.sh will start the java app with:
```
CMD="exec java ${JAVA_OPTIONS} \
-Djava.library.path=\"${KOGITO_HOME}/bin/lib\" \
-jar \"${KOGITO_HOME}/bin/quarkus-run.jar\""
```

So I believe we have to remove the `quarkus-app` from the path to make it work.